addons/chatmon/chatmon.lua

```diff
@@ -47,42 +47,36 @@ local function play_sound(sound)
     end
 end
 
-local function get_match(match)
-    if match ~= '<name>' then
-        return match
-    end
-
-    -- this is done to have parity with the old plugin.
-    local player_name = windower.ffxi.get_player().name:lower()
-    return '* ' .. player_name .. '|'
-                .. player_name .. ' *|*\''
-                .. player_name .. '\'*|*('
-                .. player_name .. ')*|'
-                .. player_name .. '|* '
-                .. player_name .. ' *|* '
-                .. player_name .. '? *|* '
-                .. player_name .. '?|'
-                .. player_name .. '? *|'
-                .. player_name .. '?|*<'
-                .. player_name .. '>*'
-end
-
-local function check_triggers(from, text)
+local function check_triggers(from, text, sender)
     if windower.has_focus() and settings.DisableOnFocus then
         return
     end
 
+    text = windower.convert_auto_trans(text)
+    text = text:strip_color()
```

Comment:
`:strip_format` is more appropriate, but depending on the text source it may not make a difference, but still the more correct generic solution.

Reply:
It can - if the text is evaluated with the auto-translate code resolved, `convert_auto_trans` does nothing anymore, but it will contain the auto-translate braces. To cover every use case, you need both `convert_auto_trans` and `strip_format`.